v8/gruberwine/gruberwine.theme

```diff
@@ -0,0 +1,8 @@
+[Theme]
+engine = jinja
+parent = base-jinja
```

Comment:
How many files are shared between zen-jinja and gruberwine? If there are many duplicates, it might be better to inherit from zen-jinja and remove the duplicates.

Reply:
OK, I've reviewed everything and removed several templates that I had imported but not updated. I also removed some libraries that don't belong in templates.

As to the inheritance: although I say it depends on zen-jinja, that's mainly visual. I started from the zen-jinja CSS, but changed it quite a lot. My inclination is to inherit from as high up the tree as possible, but I'm happy to be guided by convention if that's not right.